automated/utils/test-runner.py

```diff
@@ -772,6 +785,43 @@ def send_to_lava(self, data):
         self.logger.debug('lava-run: cmd: {}'.format(cmd))
         subprocess.call(shlex.split(cmd))
 
+    def get_token_from_netrc(self):
+        netrc_local = netrc.netrc()
+        authTokens = netrc_local.authenticators(f"{self.qa_reports_server}")
```

Comment:
this might fail because .netrc usually doesn't contain schema. You might need to use urlsplit to work around that. urlsplit also helps deal with trailing slashes.

Reply:
Fixed it.